app/models/bookable_slot.rb

```diff
@@ -8,26 +8,26 @@ def self.regenerate_for_six_weeks
   end
 
   def self.within_date_range(from, to)
-    where('usable_slots.start_at > ? AND usable_slots.end_at < ?', from, to)
+    where('bookable_slots.start_at > ? AND bookable_slots.end_at < ?', from, to)
```

Comment:
Ah, yeah if they're chained that is an issue. Perhaps we should use `quoted_table_name` in place of the explicit table naming then... I'm merging this for now anyway.